src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+
+    private final Ui ui;
+    private final TaskList taskLst;
+    private Storage storage;
+
+    /**
+     * The constructor of Duke. Components needed to run Duke should be initialized here.
+     */
+    public Duke() {
+        ui = new Ui();
+        taskLst = new TaskList();
```

Comment:
Yeap, that could be better. The original intention of naming it `taskLst` was to signify that this variable accepts methods that `List` exposes. It would probably be more useful in an editor environment that does not provide auto-completion for methods.